### PR TITLE
Fix Electron 9.4 frame ipc bug

### LINKED ISSUE #1881: Initial cluster dashboard open is blank
**Describe the bug**

After #1799 a cluster dashboard stays blank if it's automatically opened when Lens starts. If window is reloaded completely then a cluster dashboard starts to show. Probably something to do with Electron 9.4 / IPC.

### LINKED ISSUE #1882: Reload command doesn't work in cluster dashboard
While cluster dashboard is opened, reload command has no effect. Hovewer, it works on "internal" pages without active cluster iframe.

### PR DESCRIPTION
**Pull request by jakolehm**
Fixes #1882
Fixes #1881

### CODE REVIEW

File: src/renderer/bootstrap.tsx [hunk 1]
```diff
@@ -70,4 +70,6 @@ export async function bootstrap(App: AppComponent) {
 }
 
 // run
-bootstrap(process.isMainFrame ? LensApp : App);
+setTimeout(() => {
```
Comment (jakolehm) [inline src/renderer/bootstrap.tsx]: Not needed, will cleanup.

File: src/renderer/components/app.tsx [hunk 2]
```diff
@@ -99,6 +99,7 @@ export class App extends React.Component {
     }
 
     reaction(() => this.warningsCount, (count) => {
+      console.log("warnings", count);
```
Comment (nevalla) [inline src/renderer/components/app.tsx]: Is this intentional?

**Review by nevalla — DISMISSED** [review]

File: src/renderer/components/app.tsx (hunk 2 above)
Comment (jakolehm) [reply src/renderer/components/app.tsx]: No 🤦

**Review by ixrock — DISMISSED** [review]
LGTM

File: src/renderer/components/app.tsx (hunk 2 above)
Comment (jakolehm) [reply src/renderer/components/app.tsx]: Fixed.

**Review by nevalla — APPROVED** [review]

**Comment by jakolehm** [comment]
@ixrock ptal again.

**Review by chenhunghan — COMMENTED** [review]
LTGM

**Review by chenhunghan — APPROVED** [review]

**Pull request merged**